### ISSUE TITLE
Show date and tags in the lists of articles.

### ISSUE THREAD

**Issue by nichtich** [openjournals/brief-ideas#108]
The lists of articles should include date and tags next to the number of likes.

**Comment by DLFinney** [comment]
Yes, definitely would be useful to see the tags, and be able to easily narrow down the results by clicking on them.
